conf.py

```diff
@@ -55,7 +54,7 @@
 # Add any paths that contain custom themes here, relative to this directory.
 
 # Links
-ros1_distro = "noetic"
+ros1_distro = "foxy"
```

Comment:
Should we change this to `ros2_distro` and replace it everywhere in tutorials (can be a separate issue)

Reply:
I agree, or simply ros_distro